IntroScenes/FirstStoryCutScene.gd

```diff
@@ -31,226 +31,119 @@ func _ready():
 #func _process(delta):
 #	pass
 func d1():
-	newdialog([
-	"10 years ago...",
-	TrainerName + "'s family lived in Bealbeach City,",
-	"the largest city in Tandor."
-	], true)
+	newdialog(tr("CUTSCENE_INTRO_D1"), true)
 	dialog.rect_position = MIDDLE
 	pass
 	
 func d2():
-	newdialog([
-	"Their father, Kellyn, was a brave",
-	"and talented Pokémon Ranger."
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D2"), false)
 	dialog.rect_position = BOTTOM
 	pass
 func d3():
-	newdialog([
-	"Their mother, Lucille, was a brilliant",
-	"scientist and an expert on Nuclear Energy."
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D3"), false)
 	dialog.rect_position = BOTTOM
 	pass
 func d4():
-	newdialog([
-	"Although they both worked demanding jobs,",
-	"they always found time to spend at home",
-	"with young " + TrainerName + "."
-	], true)
+	newdialog(tr("CUTSCENE_INTRO_D4"), true)
 	dialog.rect_position = MIDDLE
 	pass	
 func d5():
-	newdialog([
-	"The Tandor region was at peace,",
-	"and Lucille's company was working on a form",
-	"of clean energy that would change the world"
-	], true)
+	newdialog(tr("CUTSCENE_INTRO_D5"), true)
 	dialog.rect_position = MIDDLE
 	pass
 func d6():
-	newdialog([
-	"The future seemed bright.",
-	"Until..."
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D6"), false)
 	dialog.rect_position = MIDDLE
 	pass
 func d7():
-	newdialog([
-	"It was an ordinary day at the Nuclear",
-	"Power Plant where Lucille worked."
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D7"), false)
 	dialog.rect_position = MIDDLE
 	pass
 func d8():
-	newdialog([
-	"As part of a routine inspection,",
-	"they brought in a specialist to take",
-	"a look at the electrical system."
-	], false)
+	newdialog(tr("CUTSCENE_INTRO_D8"), false)
```

Comment:
I think you missed some of the `newdialog`. Should be `new_dialog`

Reply:
Fixed